tests/test_velocity_verlet_integrator.py

```diff
@@ -107,13 +107,12 @@ def test_reversibility():
         # range of values in the velocities. Setting the seed
         # here keeps the range the same for all n_step values.
         np.random.seed(seed)
-        lamb_sched = np.linspace(0, 1, n_steps)
```

Comment:
Observation (no action needed) -- This does weaken the property being tested, but that's fine -- I don't think we were relying on this property.